src/cmd.fs

```diff
@@ -173,30 +181,36 @@ module Cmd =
                    (ofSuccess: _ -> 'msg)
                    (ofError: #exn -> 'msg) : Cmd<'msg> =
             let bind dispatch =
-                (task arg)
-                    .``then``(ofSuccess >> dispatch)
-                    .catch(unbox >> ofError >> dispatch)
-                    |> ignore
+                try
+                    (task arg)
+                        .``then``(ofSuccess >> dispatch)
+                        .catch(unbox >> ofError >> dispatch)
+                        |> ignore
+                with x -> x |> unbox |> ofError |> dispatch
```

Comment:
Here I followed the pattern from the existing implementation of using `unbox`.

This seems to be needed because of the `#exn` condition, when others Cmd (outside of `Promise`) don't have that same requirements they only rely on `exn` and not `#exn`.